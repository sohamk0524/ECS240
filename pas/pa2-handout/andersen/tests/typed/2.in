c Test case with only address-of statements.
p 6 4
t 1 1
t 2 1
t 3 1
t 4 0
t 5 0
t 6 0
c layer 1
s 0 1 -1 4
s 0 2 -1 4
s 0 3 -1 5
s 0 3 -1 6
