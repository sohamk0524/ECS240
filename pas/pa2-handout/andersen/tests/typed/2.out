pt 1 4
pt 2 4
pt 3 5
pt 3 6
